src/asvz_bot.py

```diff
@@ -563,12 +599,14 @@ def __organisation_login_default(self, driver):
 
     def __wait_for_free_places(self, driver):
         while True:
-            try:
-                driver.find_element(
-                    By.XPATH,
-                    "//div[@class='alert alert-warning'][contains(., 'ausgebucht')]",
-                )
-            except NoSuchElementException:
+            # Instead of checking alert banner look the amount of free places
```

Comment:
Nitpick: This comment makes sense with the diff of this PR in mind, but not afterward 🙃

```suggestion
```